systemtest/src/main/java/io/strimzi/systemtest/Constants.java

```diff
@@ -240,4 +240,9 @@
      * Tag for tests where cruise control used
      */
     String CRUISE_CONTROL = "cruisecontrol";
+
+    /**
+     * Tag for tests where mainly dynamic configuration is used
+     */
+    String DYNAMIC_CONFIGURATION = "dynamicconfiguration";
```

Comment:
I think that @im-konge is in other PR adding the tag `ROLLING_UPDATE`. I think we should use it here as well because these are closely related.